sources/platform/integrations/ai/langchain.md

```diff
@@ -126,7 +126,7 @@ LangChain is a standard interface through which you can interact with a variety
 
 ## Resources
 
-- <https://python.langchain.com/docs/get_started/introduction>
-- <https://python.langchain.com/docs/integrations/providers/apify>
-- <https://python.langchain.com/docs/integrations/tools/apify>
-- <https://python.langchain.com/docs/modules/model_io/llms/>
+- [https://python.langchain.com/docs/get_started/introduction](https://python.langchain.com/docs/get_started/introduction)
```

Comment:
do we really need to do this? i thought this is enough
```suggestion
- https://python.langchain.com/docs/get_started/introduction
```

Reply:
Uh oh, apparently the markdown linter in the CI checks [doesn't like bare URLs](https://github.com/apify/apify-docs/actions/runs/10679542720/job/29599005818?pr=1176). 

And the apparent fix is to [enclose it in angle brackets](https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md), isn't that convenient 🙃 

I'm all for bare URLs; you're right that writing the same URL twice makes it less readable (i.e., not really what using linter should do). I'll check what we can do to disable this rule in markdownlint - is that okay with you @TC-MO ?